### ISSUE TITLE
Uncaught Error: Could not find module `ember-get-config` imported from `myapp/initializers/browser/leaflet-assets`

### ISSUE THREAD

**Issue by steveclarkin** [miguelcobain/ember-leaflet#140]
I recently updated to Ember 2.12.0 and started getting the following error:

`Uncaught Error: Could not find module `ember-get-config` imported from `myapp/initializers/browser/leaflet-assets`
    at missingModule (loader.js:228)
    at findModule (loader.js:239)
    at Module.findDeps (loader.js:173)
    at findModule (loader.js:243)
    at requireModule (loader.js:26)
    at resolveInitializer (index.js:3)
    at registerInitializers (index.js:16)
    at exports.default (index.js:42)
    at Module.callback (app.js:13)
    at Module.exports (loader.js:114)`

I noticed that ember-leaflet 3.0.9 depends on ember-get-config 0.2.0. If I update ember-get-config to 0.2.1, the issue appears resolved.

**Comment by miguelcobain** [comment]
I published version `3.0.10` version which should be compatible with ember 2.12. Thank you for reporting.

**Comment by muziejus** [comment]
Weirdly, I'm getting this error again today:

```
Uncaught Error: Could not find module `ember-get-config` imported from `(require)`
    at missingModule (loader.js:247)
    at findModule (loader.js:258)
    at requireModule (loader.js:24)
    at eval (ember-get-config.js:1)
    at Object.../externals/ember-get-config.js (chunk.bb72f027e0f3863f83da.js:1521)
    at __webpack_require__ (chunk.51f51e323db24546eaee.js:31)
    at eval (leaflet-assets.js:7)
    at Object../node_modules/ember-leaflet/initializers/leaflet-assets.js (chunk.dcb52b452f126908dff6.js:3081)
    at __webpack_require__ (chunk.51f51e323db24546eaee.js:31)
    at eval (leaflet-assets.js:6)
```

This is in the console, not in the build, which works fine. I've installed `ember-get-config` into my app, and the version matches the version in `ember-leaflet`'s `package.json`, but still nothing.

Ember 3.28.0 w/ Embroider (in case there is a building error)
